cms/domain/beneficiary.py

```diff
@@ -0,0 +1,17 @@
+import dataclasses
+
+@dataclasses.dataclass
+class Beneficiary:
+    beneficiary_id: str #TODO- Currently nanoid, move to some ID class
+    fname: str
+    lname: str
+    mname: str
+    phone: str
+    email: str
```

Comment:
You can add govt_id and use cms.domain.id_type import IDType

Reply:
can we add govt id in next checkin? 
there were some sytacx issues when enum was referred for second time. 
I wanted to get 2nd table added to understand the code structure first.